.github/workflows/release.yaml

```diff
@@ -9,12 +9,35 @@ jobs:
   release:
     runs-on: ubuntu-latest
     timeout-minutes: 60
+    strategy:
+      matrix:
+        node-version: ['lts/*']
+      fail-fast: false
     steps:
-      - name: Checkout
+      - name: Checkout project
         uses: actions/checkout@v3
 
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache Node dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: npm install & build
+        run: |
+          npm install
+          npm run build --if-present
+
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
-          token: ${{ secrets.PAT }}
+          # token: ${{ secrets.PAT }}
```

Comment:
Oops...probably should remove this. Do we have a secret named `PAT` setup already? I didn't set a PAT secret in my fork and just let it use the default token provided by actions.

Reply:
If the default token works we should use that, otherwise I'll create a PAT secret with the right scope.

Reply:
I've pulled it out completely for now. If we find we need our own PAT we can do that later